src/test/java/jraft/impl/RaftServerContextImplTest.java

```diff
@@ -1,18 +1,34 @@
 package jraft.impl;
 
 import jraft.RaftServerContext;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.*;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
 
+import java.rmi.server.ExportException;
```

Comment:
be careful if this Exception is something you actually want or not? it's weird to use from rmi...